tools/release/release_notes.py

```diff
@@ -86,6 +86,40 @@
 API_URL = "https://api.github.com/repos/grpc/grpc/pulls/"
 
 
+def print_commits_wo_pr(commits_wo_pr):
```

Comment:
This method is a no-op if `commits_wo_pr` is empty. Can we just not call the function instead of making the repeated `if commits_wo_pr` checks?

Reply:
Done.